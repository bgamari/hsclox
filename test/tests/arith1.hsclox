print (1+2*3);
print ((2 + 3) / 5);
print -(-1);
