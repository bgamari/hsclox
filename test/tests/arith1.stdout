== code ==
0000    1 OP_CONSTANT         0 '1
0002    | OP_CONSTANT         1 '2
0004    | OP_CONSTANT         2 '3
0006    | OP_MULTIPLY
0007    | OP_ADD
0008    2 OP_RETURN



0000    1 OP_CONSTANT         0 '1

[ 1 ]

0002    | OP_CONSTANT         1 '2

[ 1 ][ 2 ]

0004    | OP_CONSTANT         2 '3

[ 1 ][ 2 ][ 3 ]

0006    | OP_MULTIPLY

[ 1 ][ 6 ]

0007    | OP_ADD

[ 7 ]

0008    2 OP_RETURN
7
